src/appleseed-max-impl/appleseedobjpropsmod/appleseedobjpropsmod.cpp

```diff
@@ -145,7 +146,10 @@ namespace
         ParamIdSSSSet, L"sss_set", TYPE_STRING, 0, IDS_SSS_SET,
             p_ui, ParamMapIdVisibility, TYPE_EDITBOX, IDC_SSS_SET,
         p_end,
-
+        paramIdNestedmediumPriority, L"nested_medium_priority", TYPE_INT, P_TRANSIENT, 0,
```

Comment:
Similarly here, `medium_priority` ought to be enough.

Reply:
agree